lib/fluent/plugin/out_elasticsearch.rb

```diff
@@ -146,7 +147,7 @@ def initialize(retry_stream)
     config_param :sniffer_class_name, :string, :default => nil
     config_param :selector_class_name, :string, :default => nil
     config_param :reload_after, :integer, :default => DEFAULT_RELOAD_AFTER
-    config_param :content_type, :enum, list: [:"application/json", :"application/x-ndjson"], :default => :"application/json",
+    config_param :content_type, :enum, list: [:"application/json", :"application/x-ndjson", :"#{ES9_CONTENT_TYPE}"], :default => :"application/json",
```

Comment:
Could you tell me why `ES9_CONTENT_TYPE` is added to this deprecated param?

It looks to me that a user does not need to set this param since the `Content-Type` and `Accept` headers are automatically fixed when the ES version is greater than or equal to `9`.
Setting this value explicitly to `ES9_CONTENT_TYPE` would cause `@accept` to be `nil`.
Is this intentional?

Reply:
Oh, that's only left things for tryal-and-error investigation process.
I'll remove it from there.

Reply:
I see!